exercises/nth-prime/tests/nth-prime.rs

```diff
@@ -0,0 +1,30 @@
+extern crate nth_prime as np;
+
+#[test]
+fn test_first_prime() {
+    assert_eq!(np::nth(1), Ok(2));
+}
+
+#[test]
+#[ignore]
+fn test_second_prime() {
+    assert_eq!(np::nth(2), Ok(3));
+}
+
+#[test]
+#[ignore]
+fn test_sixth_prime() {
+    assert_eq!(np::nth(6), Ok(13));
+}
+
+#[test]
+#[ignore]
+fn test_big_prime() {
+    assert_eq!(np::nth(10001), Ok(104743));
+}
+
+#[test]
+#[ignore]
+fn test_zeroth_prime() {
+    assert!(np::nth(0).is_err());
```

Comment:
Well, I made this change and now I want to ask what y'all think about it. I don't think I would want to encourage students to have `Result<T, E>` where `E` is `()` since then we might as well just have `Option`, right? So, it seems better to just use `is_err` here instead of `assert_eq!(np::nth(0), Err(()))` . Agreed?